adoc/TRD-SLES-SAP-HA-automation-quickstart-cloud.adoc

```diff
@@ -158,33 +161,54 @@ endif::[]
 
 == Preparing the SAP HANA media
 
-The SAP HANA Media needs to be made available so it can be accessed during the deployment.
-The SUSE SAP Automation project allows for three methods for presenting the SAP media:
+With the correct entitlement, SAP HANA media can be downloaded from the SAP Web site at https://support.sap.com/en/my-support/software-downloads.html. The SAP Media needs to be made available so it can be accessed during the deployment.
+
+The SUSE Automation for SAP Applications project allows for three methods for presenting the SAP media:
 
 . SAR file and SAPCAR executable (SAP HANA Database only)
 . Multipart exe/RAR files
 . Extracted media
 
 The different formats have some benefits and drawbacks:
 
-. The compressed archives (SAR and RAR) provide simple copy to the cloud, but a bit longer install times due to extracting it during the process
-. The uncompressed / extracted media are the fastest install, but there need a huge number of files copied to the cloud share, which also takes a lot of time in forehand as preparation.
+. The compressed archives (SAR and RAR) provide a simple copy to the cloud but a longer install time due to extracting it during the process
+. The uncompressed/extracted media are the fastest install, but more files are copied to the cloud share, which also takes time in forehand as preparation.
+
 
 // we need to agree on this and unifiy the doc
+ifeval::[ "{cloud}" == "Azure" ]
```

Comment:
what is not understandable on the comment?

Reply:
Nothing, it might be one new line added by mistake.